src/elements/callout/Callout.tsx

```diff
@@ -8,34 +11,9 @@ import type { FieldNameToValueMap } from "../../plugin/helpers/fieldView";
 import { dropDownRequired } from "../../plugin/helpers/validation";
 import { createReactElementSpec } from "../../renderers/react/createReactElementSpec";
 import { CustomDropdownView } from "../../renderers/react/customFieldViewComponents/CustomDropdownView";
-import { CalloutError, calloutStyles } from "../embed/Callout";
+import { CalloutError } from "./CalloutError";
 import { CalloutTable } from "./CalloutTable";
-
-export type Fields = {
-  callout: string;
-  formId: number;
-  tagName: string;
-  description?: string;
-  formUrl?: string;
-  _type: string;
-};
-
-export type Rules = {
-  requiredTags: string[];
-  lackingTags: string[];
-  matchAllTags: boolean;
-};
-
-export type Campaign = {
-  id: string;
-  name: string;
-  fields: Fields;
-  rules: Rules[];
-  priority: number;
-  displayOnSensitive: boolean;
-  activeFrom?: number;
-  activeUntil?: number;
-};
```

Comment:
we use these types in a few different places. In order to keep them synced, and to prevent any circular dependencies, i've extracted them into a types file within the callout folder. I couldn't find a current convention for this so let me know if this is not idiomatic to the project.